### PR TITLE
Update README.md file with easier to use instructions

### PR DESCRIPTION
**Pull request by StefanJum**
This will remove the running scripts and add the commands in the README file, rename the `config-*` files and add a `TLDR` section that summarises the setup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,52 @@
...
+This will configure, build and run the `httpreply` application.
+You can see how to test it in the [running section](#run).
+
+The same can be done for `AArch64`, by running the commands below:
+
+```console
+make properclean
+UK_DEFCONFIG=$(pwd)/.config.httpreply_qemu-arm64 make defconfig
+make -j $(nproc)
+sudo /usr/bin/qemu-system-aarch64 \
```
Comment (razvand) [inline README.md]:
```suggestion
sudo qemu-system-aarch64 \
```

File: README.md [hunk 2]
```diff
@@ -3,6 +3,52 @@
...
+git clone https://github.com/unikraft/unikraft .unikraft/unikraft
+git clone https://github.com/unikraft/lib-lwip .unikraft/libs/lwip
+UK_DEFCONFIG=$(pwd)/.config.httpreply_qemu-x86_64 make defconfig
+make -j $(nproc)
+sudo ip link set dev virbr0 down 2> /dev/null
+sudo ip link del dev virbr0 2> /dev/null
+sudo ip link add dev virbr0 type bridge
+sudo ip address add 172.44.0.1/24 dev virbr0
+sudo ip link set dev virbr0 up
+sudo /usr/bin/qemu-system-x86_64 \
```
Comment (razvand) [inline README.md]:
```suggestion
sudo qemu-system-x86_64 \
```

File: README.md [hunk 3]
```diff
@@ -239,14 +304,33 @@ This image is to be used in the run step.
 
 ## Run
 
-Run the resulting image with the `run-...` scripts.
+Run the resulting image with the `qemu-system-*` commands.
+In order to run `nginx` you need to first set up a network bridge.
```
Comment (razvand) [inline README.md]:
```suggestion
In order to run `httpreply` you need to first set up a network bridge.
```

File: README.md [hunk 4]
```diff
@@ -239,14 +304,33 @@ This image is to be used in the run step.
...
+```
 
 ### QEMU x86_64
 
-To run the QEMU x86_64 build, use `run-qemu-x86_64.sh`:
+To run the QEMU x86_64 build, use `qemu-system-x86_64`:
 
 ```console
-$ ./run-qemu-x86_64.sh
+sudo /usr/bin/qemu-system-x86_64 \
```
Comment (razvand) [inline README.md]:
```suggestion
sudo qemu-system-x86_64 \
```

File: README.md [hunk 5]
```diff
@@ -283,10 +372,19 @@ that is press the `Ctrl` and `a` keys at the same time and then, separately, pre
 
 ### QEMU AArch64
 
-To run the AArch64 build, use `run-qemu-aarch64.sh`:
+To run the AArch64 build, use `qemu-system-aarch64`:
 
 ```console
-$ ./run-qemu-aarch64.sh
+sudo /usr/bin/qemu-system-aarch64 \
```
Comment (razvand) [inline README.md]:
```suggestion
sudo qemu-system-aarch64 \
```

File: README.md [hunk 6]
```diff
@@ -239,14 +304,33 @@ This image is to be used in the run step.
 
 ## Run
 
-Run the resulting image with the `run-...` scripts.
+Run the resulting image with the `qemu-system-*` commands.
+In order to run `nginx` you need to first set up a network bridge.
+You can do that by running the following commands:
+
+```console
```
Comment (razvand) [inline README.md]: This is too complicated. I say we re-add the `run-...` scripts. This is what we have for [`app-nginx`](https://github.com/unikraft/app-nginx), and I would keep it here. Makes it easier to test.

**Review by razvand — CHANGES_REQUESTED** [review]
Hi, @StefanJum. Let's use the direct `qemu-system-...` command, not the full path.

Also, add back the `run-qemu-...` scripts that create the bridges behind the scenes.

**Comment by StefanJum** [comment]
Done @razvand

File: .config.httpreply_qemu-arm64 [hunk 7]
```diff
@@ -0,0 +1,183 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Unikraft/0.13.1~5eb820bd Configuration
+#
+CONFIG_UK_FULLVERSION="0.13.1~5eb820bd"
+CONFIG_UK_CODENAME="Atlas"
+CONFIG_UK_ARCH="arm64"
+CONFIG_UK_BASE="/media/stefan/projects/unikraft/usoc23/test-01/httpreply/.unikraft/unikraft"
+CONFIG_UK_APP="/media/stefan/projects/unikraft/usoc23/test-01/httpreply"
+CONFIG_UK_DEFNAME="httpreply"
```
Comment (razvand) [inline .config.httpreply_qemu-arm64]:
```suggestion
CONFIG_UK_ARCH="arm64"
CONFIG_UK_DEFNAME="httpreply"
```

File: .config.httpreply_qemu-x86_64 [hunk 8]
```diff
@@ -0,0 +1,181 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Unikraft/0.13.1~5eb820bd Configuration
+#
+CONFIG_UK_FULLVERSION="0.13.1~5eb820bd"
+CONFIG_UK_CODENAME="Atlas"
+CONFIG_UK_ARCH="x86_64"
+CONFIG_UK_BASE="/media/stefan/projects/unikraft/usoc23/test-01/httpreply/.unikraft/unikraft"
+CONFIG_UK_APP="/media/stefan/projects/unikraft/usoc23/test-01/httpreply"
+CONFIG_UK_DEFNAME="httpreply"
```
Comment (razvand) [inline .config.httpreply_qemu-x86_64]:
```suggestion
CONFIG_UK_ARCH="x86_64"
CONFIG_UK_DEFNAME="httpreply"
```

**Review by razvand — CHANGES_REQUESTED** [review]
@StefanJum, just some minor changes. Then we can approve it.

**Comment by StefanJum** [comment]
Done @razvand

**Review by razvand — APPROVED** [review]
Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request closed without merging**